File: trail1/윤년인가/is-leap-year.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        int Y = int.Parse(Console.ReadLine());
        string res = ((Y % 4 == 0 && Y % 100 != 0) || Y % 400 == 0) ? "true" : "false";
        Console.WriteLine(res); 
    }
}

File: trail1/윤년인가/is-leap-year.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    int Y;
    cin >> Y;
    string res = (Y % 4 == 0 && Y % 100 != 0) || Y % 400 == 0 ? "true" : "false";
    cout << res << " ";    
    return 0;
}